test/server.cpp

```diff
@@ -296,6 +295,10 @@ const char* urls404[] = {
   "/ROOT/suggest?content=non-existent-book&term=abcd",
   "/ROOT/catch/external",
   "/ROOT/zimfile/A/non-existent-article",
+
+  // zimfile has no Favicon nor Illustration_48x48@1 meta item
+  "/ROOT/raw/zimfile/meta/Favicon",
+  "/ROOT/raw/zimfile/meta/Illustration_48x48@1",
```

Comment:
This suggestion should be made in #646, but I only reviewed the changes and didn't think about testing of the HTTP 404 error functionality (which didn't show up in the diff). It makes sense to test how the `/raw` endpoint reacts to
- non-existent ZIM file
- invalid `kind` (neither `meta` nor `content`)